src/OpcApplicationConfiguration.cs

```diff
@@ -121,6 +121,9 @@ public async Task<ApplicationConfiguration> ConfigureAsync()
             .SetMaxRequestThreadCount(MAX_REQUEST_THREAD_COUNT)
             // LDS registration interval
             .SetMaxRegistrationInterval(LdsRegistrationInterval);
+            // enable auditing events and diagnostics
+            .SetDiagnosticsEnabled(true)
+            .SetAuditingEnabled(true)
 

```

Comment:
@koepalex use fluent api
@luiscantero worth some config options?

Reply:
If it generates a lot of log output, it might be good if it's optional. Does it?

Reply:
diagnostics only produces output with the service calls, audit events only produce events. Logging should not be affected.